.github/workflows/build.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         if: steps.cache-pact.outputs.cache-hit != 'true'
         run: |
           cd ~
-          curl -fsSL https://raw.githubusercontent.com/pact-foundation/pact-ruby-standalone/master/install.sh | bash
+          curl -fsSL https://raw.githubusercontent.com/pact-foundation/pact-ruby-standalone/master/install.sh | PACT_CLI_VERSION=v2.4.2 sh
```

Comment:
Could we replace this line and the one below with `make tools` to consolidate the pact version in one place (the makefile)?

Just a non-blocking thought.